wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -248,7 +248,7 @@ static function enqueue_scripts() {
 			)
 		);
 
-		if( $current_screen->id === 'upload' || $current_screen->id === 'post' ||
+		if( $current_screen->id === 'upload' || $pagenow === 'post.php' ||
```

Comment:
`post.php` is the general page for editing any kind of post...`$current_screen->id` === post type slug. I inspected `$current_screen` and didn't see anything that would definitively show we were on any post edit page, so I think `$pagenow` is better for this. Also you did write this 😼#thxgitblame